sklearn/model_selection/_search.py

```diff
@@ -552,7 +562,26 @@ def fit(self, X, y=None, groups=None):
         """
         estimator = self.estimator
         cv = check_cv(self.cv, y, classifier=is_classifier(estimator))
-        self.scorer_ = check_scoring(self.estimator, scoring=self.scoring)
+
+        self.scorer_, self.multimetric_ = _check_multimetric_scoring(
+            self.estimator, scoring=self.scoring)
+
+        if self.multimetric_:
+            if self.refit and (
```

Comment:
hmm. This will break if `self.refit` is the empty string. Should we validate against that case? Or allow it?